photon-ml/src/main/scala/com/linkedin/photon/ml/algorithm/CoordinateDescent.scala

```diff
@@ -14,42 +14,41 @@
  */
 package com.linkedin.photon.ml.algorithm
 
-import scala.collection.Map
 import org.apache.spark.rdd.RDD
 
 import com.linkedin.photon.ml.{BroadcastLike, RDDLike}
 import com.linkedin.photon.ml.constants.{StorageLevel, MathConst}
 import com.linkedin.photon.ml.data.{DataSet, GameDatum}
 import com.linkedin.photon.ml.evaluation.Evaluator
-import com.linkedin.photon.ml.model.Model
+import com.linkedin.photon.ml.model.GAMEModel
 import com.linkedin.photon.ml.util.{ObjectiveFunctionValue, PhotonLogger}
 
 /**
- * Coordinate descent implementation
- *
- * @param coordinates the individual optimization problem coordinates. The coordinates is a [[Seq]] consists of
- *                    (coordinateName, [[Coordinate]] object) pairs.
- * @param trainingLossFunctionEvaluator training loss function evaluator
- * @param validatingDataAndEvaluatorOption optional validation data evaluator. The validating data is a [[RDD]] consists
- *                                         of (global Id, [[GameDatum]] object pairs), there the global Id is a unique
- *                                         identifier for each [[GameDatum]] object.
- * @param logger logger instance
- * @author xazhang
- */
+  * Coordinate descent implementation
+  *
+  * @param coordinates the individual optimization problem coordinates. The coordinates is a [[Seq]] consists of
+  *                    (coordinateName, [[Coordinate]] object) pairs.
+  * @param trainingLossFunctionEvaluator training loss function evaluator
+  * @param validatingDataAndEvaluatorOption optional validation data evaluator. The validating data is a [[RDD]]
+  *                                         consists of (global Id, [[GameDatum]] object pairs), there the global Id
+  *                                         is a unique identifier for each [[GameDatum]] object.
+  * @param logger logger instance
+  * @author xazhang
+  */
```

Comment:
Hah! I've never noticed that the Scala comment block style differs from the Java standard like this.